.travis.yml

```diff
@@ -59,16 +59,14 @@ jobs:
       -Dmaven.javadoc.skip=true
     - bash <(curl -s https://codecov.io/bash)
     - .travis_scripts/javadocs.sh
-
+    
 notifications:
   email:
-  - corfudb-dev@googlegroups.com
-  webhooks:
-    urls:
-    - https://webhooks.gitter.im/e/e26458f01e3586e3c140
-    on_success: always
+    recipients:
```

Comment:
is this only for the master branch?

Reply:
I think so.

> Pull Request builds do not trigger email notifications.



https://docs.travis-ci.com/user/notifications/#configuring-email-notifications